content/en/load_balancer.md

```diff
@@ -3,12 +3,15 @@ title: Load Balancer
 status: Feedback Appreciated
 category: concept
 ---
-
 ## What it is
-A load balancer is a method to distribute incoming network traffic across a group of servers in the back-end. The solution can be software or hardware based.
+
```

Comment:
For anyone's information:
it is good to insert empty lines before and after the headers, like this. 😊

- https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md022---headers-should-be-surrounded-by-blank-lines
- https://arcticicestudio.github.io/styleguide-markdown/rules/headings.html#empty-lines-before-and-after

Reply:
Yes, I'll do that. Thanks

Reply:
If this is resolved, please mark it as such.